### ISSUE TITLE
Rector\Renaming\Rector\MethodCall\RenameMethodCallRector replacing wrong method

### ISSUE THREAD

**Issue by craigh** [rectorphp/rector#2615]
| Subject        | Details                                                      |
| :------------- | :----------------------------------------------------------- |
| Rector version | v0.6.9         |
| PHP version    | PHP 7.3.9                               |
| Full Command   | `src/bin/rector process src/lib/Zikula/Bundle/FormExtensionBundle/Form/Type/InlineFormDefinitionType.php --dry-run --debug`
  | 

## Current Behaviour 
```yaml
# rector.yaml
services:
  # change `__()` and `__f()` to `trans()`
  Rector\Renaming\Rector\MethodCall\RenameMethodCallRector:
    $oldToNewMethodsByClass:
      Zikula\Common\Translator\TranslatorInterface:
        __: 'trans'
        __f: 'trans'
```

```powershell
[Craigs-iMac2013:MAMP/htdocs/core.git] craig% src/bin/rector process src/lib/Zikula/Bundle/FormExtensionBundle/Form/Type/InlineFormDefinitionType.php --dry-run --debug
File "/Applications/MAMP/htdocs/core.git/src/vendor/rector/rector/../../autoload.php" is about to be loaded in "AutoloadIncluder::autoloadProjectAutoloaderFile()" on line 97"
Rector v0.6.9
Config file: rector.yaml

[parsing] src/lib/Zikula/Bundle/FormExtensionBundle/Form/Type/InlineFormDefinitionType.php
[refactoring] src/lib/Zikula/Bundle/FormExtensionBundle/Form/Type/InlineFormDefinitionType.php
    [applying] Rector\Renaming\Rector\MethodCall\RenameMethodCallRector
    [applying] Rector\Renaming\Rector\MethodCall\RenameMethodCallRector
    [applying] Rector\Renaming\Rector\MethodCall\RenameMethodCallRector
    [applying] Rector\Renaming\Rector\MethodCall\RenameMethodCallRector
    [applying] Rector\Renaming\Rector\MethodCall\RenameMethodCallRector
    [applying] Rector\Renaming\Rector\MethodCall\RenameMethodCallRector
    [applying] Rector\Renaming\Rector\MethodCall\RenameMethodCallRector
    [applying] Rector\Renaming\Rector\MethodCall\RenameMethodCallRector
    [applying] Rector\Renaming\Rector\MethodCall\RenameMethodCallRector
    [applying] Rector\Renaming\Rector\MethodCall\RenameMethodCallRector
    [applying] Rector\Renaming\Rector\MethodCall\RenameMethodCallRector
    [applying] Rector\Renaming\Rector\MethodCall\RenameMethodCallRector
[printing] src/lib/Zikula/Bundle/FormExtensionBundle/Form/Type/InlineFormDefinitionType.php


1 file with changes
===================

1) src/lib/Zikula/Bundle/FormExtensionBundle/Form/Type/InlineFormDefinitionType.php

    ---------- begin diff ----------
--- Original
+++ New
@@ -57,7 +57,7 @@
 
         foreach ($this->dynamicFieldsContainer->getDynamicFieldsSpecification() as $fieldSpecification) {
             $fieldOptions = $fieldSpecification->getFormOptions();
-            $fieldOptions['label'] = $fieldOptions['label'] ?? $fieldSpecification->getLabel($this->translator->getLocale());
+            $fieldOptions['label'] = $fieldOptions['label'] ?? $fieldSpecification->getLabel($this->translator->trans());
             $this->removeChoiceLoader($fieldSpecification->getFormType(), $fieldOptions);
 
             $prefix = $fieldSpecification->getPrefix();
    ----------- end diff -----------

Applied rules:

 * Rector\Renaming\Rector\MethodCall\RenameMethodCallRector


                                                                                                                        
 [OK] Rector is done! 1 changed files                                                                                   
```

## original
[original file is here](https://github.com/zikula/core/blob/translation/src/lib/Zikula/Bundle/FormExtensionBundle/Form/Type/InlineFormDefinitionType.php#L60)

## the problem
If it isn't obvious, I am trying to replace method `__` and `__f` with method `trans`. In the example, it is replacing the `getLocale` method with `trans`, so that's wrong. 😄 In other areas, it is replacing `setDomain` with `trans`

**Comment by TomasVotruba** [comment]
Interesting :sweat_smile: 

Could you send failing test case to `RenameMethodCallRectorTest`?

**Comment by craigh** [comment]
I'm sorry, I doubt that I'll be able to do so.

**Comment by TomasVotruba** [comment]
Test helps fix it quickly.

All you need to do is duplicate one file here,
https://github.com/rectorphp/rector/tree/packages/Renaming/tests/Rector/MethodCall/RenameMethodCallRector/Fixture 
and add your content

**Comment by TomasVotruba** [comment]
Rector thought it is a regex pattern :D 

Solved by #2639
